BoredApp/View/UIView+Extension.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  UIView+Extension.swift
+//  BoredApp
+//
+//  Created by Матвей Бойков on 22.09.2021.
+//
+
+import Foundation
+import UIKit
+
+
+extension UIView {
+    
+    func round(by radius : Double = 1.0, theseCorners chosenCorners: UIRectCorner...) {
+
+        let corners : UIRectCorner = .init(chosenCorners)
+        let maskPath = UIBezierPath(roundedRect: self.bounds,
+                                     byRoundingCorners: corners,
+                                     cornerRadii: CGSize(width: radius, height: radius))
+        let maskLayer = CAShapeLayer()
+        maskLayer.frame = self.bounds
+        maskLayer.path = maskPath.cgPath
+        self.layer.mask = maskLayer
+        
+    }
+}
+
+class Colourist {
```

Comment:
в проде скорее всего будет так
тк енам легче поддерживать, нежели строки
```suggestion
enum FilterType: String {
    case education, recreational, social
    
    var color: UIColor {
        switch self {
        case .education:
            return UIColor(red: 235/255, green: 87/255, blue: 87/255, alpha: 1)
        case .recreational:
            return UIColor(red: 255/255, green: 147/255, blue: 0/255, alpha: 1)
        case .social:
            return UIColor(red: 25/255, green: 67/255, blue: 72/255, alpha: 1)
        }
    }
}
```